ClinGen/clincoded
# Connects to #450. Demo site styles

* Change styling on demo notice bar
* Make notice bar unclosable
* Add background to demo site

Testing:
* Make sure above changes are visible on local and test instances
* Make sure above changes are *not* visible on http://curation.clinicalgenome.org/ (use hosts file trick outlined in #353)

![image](https://cloud.githubusercontent.com/assets/4326866/10951399/47e1454e-82f2-11e5-9231-77d839ca496b.png)
